src/bin-install.ts

```diff
@@ -14,15 +14,21 @@ typings install [<name>=]<location>
   <location>  The location to read from (described below)
 
 Valid Locations:
-  [<source>!]<pkg>[@<version>][#<tag>]
+  <pkg>[@<version>][#<tag>]
+    Install from "defaultSource" specified in `.typingsrc`
+      default to "npm" for non-ambient package
+      default to "dt" for ambient package
+  <source>!<pkg>[@<version>][#<tag>]
+    Install from registry mirror or DefinitelyTyped (see <source> below)
```

Comment:
It seemed clear before, huh. It actually did mention the `source`, can we not just add information about the defaults down the bottom where we describe `<source>`?